editor.planx.uk/src/hooks/useSearch.ts

```diff
@@ -4,6 +4,7 @@ import { useEffect, useMemo, useState } from "react";
 interface UseSearchProps<T extends object> {
   list: T[];
   keys: Array<FuseOptionKey<T>>;
+  searchType?: "word-match" | "letter-match"
```

Comment:
```suggestion
  searchType?: "include-match" | "fuzzy-match"
```

nit: This is a bit more accurate I think.